src/ontology/mondo-ingest.Makefile

```diff
@@ -218,12 +212,18 @@ $(REPORTDIR)/%_excluded_terms_in_mondo_xrefs.tsv $(REPORTDIR)/%_excluded_terms_i
 	--component-signature-path $(REPORTDIR)/component_signature-$*.tsv \
 	--outpath $@
 
-# Exclusions: combined
-$(REPORTDIR)/term_exclusions.txt $(REPORTDIR)/exclusion_reasons.robot.template.tsv: $(foreach n,$(ALL_COMPONENT_IDS), $(REPORTDIR)/$(n)_term_exclusions.txt)
-	cat $(REPORTDIR)/*_term_exclusions.txt > $(REPORTDIR)/term_exclusions.txt; \
+# Exclusions: all artefacts for single ontology
+exclusions-%:
```

Comment:
@matentzn We had a goal for running (i) all exclusion artefacts of a certain type for all ontologies, but not a goal for running (ii) all exclusion artefacts for a single ontology, which is something I found useful today.

Actually, we are still lacking (iii) all exclusion artefacts for all ontologies. Maybe something to add.